.github/workflows/ci.yml

```diff
@@ -119,6 +119,7 @@ jobs:
         with:
           install: '${{ matrix.projectName }}...'
           build: 'false'
+          pull-playwright-cache: ${{ matrix.testEnv.shouldCreate }}
```

Comment:
An improvement here would be to only run for e2e tests, they're the only ones using the browsers. We have the `matrix.testType` for that.

```suggestion
          pull-playwright-cache: ${{ matrix.testType == 'e2e'}}
```

Reply:
Nice! But when we spinoff test envs, Playwright installs chromium for both unti/e2e tests Here example for a unit test job:

![Screenshot from 2024-07-03 13-58-34](https://github.com/woocommerce/woocommerce/assets/1577185/d3629053-cdcc-49d5-aa54-0a0781b16067)

Reply:
I'll try to fix using env:dev vs env:test for running for unit and e2e tests accordingly.

Reply:
Let see how it works: 9c21959fde82fd6d064592ce1ec6d4619d4237bb

Reply:
Ah, you're right, I spoke too soon. I fixed that spinoff test envs part in [another PR](https://github.com/woocommerce/woocommerce/pull/49017), but it's not yet merged. I basically moved the playwright install from the env setup to the test run command.
But this switch to `env:dev` will also help.

Reply:
Great! With that perspective, I think I need to revert my changes switching to `env:dev` - `env:test` reads as a better fit.

Reply:
Your call. `env:dev` will work just as well, but I agree `env:test` reads better. This reminds me that we really need to audit and clean these pnpm scripts...